helpers/callBind.js

```diff
@@ -8,7 +8,7 @@ var $apply = GetIntrinsic('%Function.prototype.apply%');
 var $call = GetIntrinsic('%Function.prototype.call%');
 var $reflectApply = GetIntrinsic('%Reflect.apply%', true) || bind.call($call, $apply);
 
-module.exports = function callBind() {
+module.exports.call = function callBind() {
```

Comment:
unfortunately this ends up being a breaking change for the many packages that directly import this helper.